src/api/index.js

```diff
@@ -6,6 +6,7 @@
 import merge from 'lodash/object/merge'
 import Emitter from 'component-emitter'
 
+import {version} from '../../package.json'
```

Comment:
this will bundle entire package.json file, use https://github.com/kentcdodds/babel-plugin-macros

Reply:
also, you have to make sure that you increase the version first, before creating the build